podcasts/Onboarding/Plus/PlusButtonStyles.swift

```diff
@@ -2,13 +2,19 @@ import SwiftUI
 
 struct PlusGradientFilledButtonStyle: ButtonStyle {
     let isLoading: Bool
-    let background: any View
-    let foregroundColor: Color
+    let plan: Constants.Plan
 
-    init(isLoading: Bool = false, background: any View = Color.plusGradient, foregroundColor: Color = .plusButtonFilledTextColor) {
+    private var background: any View {
+        plan == .plus ? Color.plusGradient : Color(hex: "6046F5")
```

Comment:
Can we add the new color here to the Color extension since it's being used in a few places?